python/sglang/srt/layers/attention/flashinfer_backend.py

```diff
@@ -408,6 +411,37 @@ def init_forward_metadata_capture_cuda_graph(
             )
             self.prefill_cuda_graph_metadata[bs] = prefill_wrappers
             self.forward_metadata = PrefillMetadata(prefill_wrappers, False, False)
+        elif forward_mode.is_extend():
```

Comment:
question: it seems this branch is similar to the draft_extend branch. is it possible we somehow unify the two

(same for other `elif` branches in this file)